third_party/2.7/requests/sessions.pyi

```diff
@@ -55,27 +55,39 @@ class SessionRedirectMixin:
 
 class Session(SessionRedirectMixin):
     __attrs__ = ... # type: Any
-    headers = ... # type: MutableMapping[str, str]
-    auth = ... # type: Any
-    proxies = ... # type: Any
-    hooks = ... # type: Any
-    params = ... # type: Any
-    stream = ... # type: Any
-    verify = ... # type: Any
-    cert = ... # type: Any
-    max_redirects = ... # type: Any
-    trust_env = ... # type: Any
-    cookies = ... # type: Any
-    adapters = ... # type: Any
-    redirect_cache = ... # type: Any
+    headers = ... # type: Optional[MutableMapping[Text, Text]]
+    auth = ... # type: Optional[Union[Tuple[Text, Text], Callable[[Request], Request]]]
```

Comment:
Out of curiosity, is there a motivation other than brevity?  I personally find `Optional[Union[x, y]]` to be a bit more clear as to the engineer's intent and more in line with optional types in other languages (although I know `mypy` translates it to `Union[None, x, y]` under the covers).